FWCore/Concurrency/test/BuildFile.xml

```diff
@@ -1,6 +1,9 @@
-<bin name="testFWCoreConcurrency" file="*.cppunit.cpp">
-  <use name="cppunit"/>
+<bin name="testFWCoreConcurrency" file="test2_catch2_*.cc">
```

Comment:
Should we eventually (after the https://github.com/cms-sw/cmssw/issues/45487 has been resolved) merge the `testFWCoreConcurrency` and `testFWCoreConcurrencyCatch2`?